runtime/storage-proxy.js

```diff
@@ -310,7 +310,7 @@ class CollectionProxy extends StorageProxyBase {
       keys = this._model.getKeys(id);
     }
     let data = {
-      id,
+      value: id,
```

Comment:
It looks like the same issue exists at line 300, should we change there as well?